test/cql-pytest/test_wasm.py

```diff
@@ -0,0 +1,396 @@
+# -*- coding: utf-8 -*-
+# Copyright 2021-present ScyllaDB
+#
+# This file is part of Scylla.
+#
+# Scylla is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Scylla is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Scylla.  If not, see <http://www.gnu.org/licenses/>.
+
+#############################################################################
+# Tests for user defined functions defined with WebAssembly backend
+#############################################################################
+
+from cassandra.protocol import InvalidRequest
+from util import new_test_table, unique_name
+
+import pytest
+
+# Test that calling a wasm-based fibonacci function works
+def test_fib(cql, test_keyspace):
+    fib_name = "fib_" + unique_name()
+    fib_source = f"""
+(module
+  (func ${fib_name} (param $n i64) (result i64)
+    (if
+      (i64.lt_s (local.get $n) (i64.const 2))
+      (return (local.get $n))
+    )
+    (i64.add
+      (call ${fib_name} (i64.sub (local.get $n) (i64.const 1)))
+      (call ${fib_name} (i64.sub (local.get $n) (i64.const 2)))
+    )
+  )
+  (export "{fib_name}" (func ${fib_name}))
+)
+"""
+    schema = 'pk bigint primary key'
+    with new_test_table(cql, test_keyspace, schema) as table:
```

Comment:
Alright, I created multiple tables because I needed them to have multiple types, but you're right that I can just use a long list of columns in a single table.